pkg/iac/terraform/state/test/vpc_security_group/result.golden.json

```diff
@@ -1,20 +1,44 @@
 [
  {
-  "Arn": "arn:aws:ec2:eu-west-3:047081014315:security-group/sg-0254c038e32f25530",
-  "Description": "Managed by Terraform",
-  "Egress": null,
   "Id": "sg-0254c038e32f25530",
-  "Ingress": null,
-  "Name": "foo",
-  "NamePrefix": "",
-  "OwnerId": "047081014315",
-  "RevokeRulesOnDelete": false,
-  "Tags": {
-   "Name": "Foo SG"
-  },
-  "VpcId": "vpc-618f6e09",
-  "Timeouts": null,
-  "CtyVal": {}
+  "Type": "aws_security_group",
+  "Attrs": {
+   "arn": "arn:aws:ec2:eu-west-3:047081014315:security-group/sg-0254c038e32f25530",
+   "description": "Managed by Terraform",
+   "egress": [
```

Comment:
Previously we got null for `egress` and `ingress`, we remove them in the normalize func, I don't get why we got them there 🤔

Reply:
hmm good question. We still safe delete them in normalize function if I'm right. I'll check...